src/lib/button-toggle/button-toggle.scss

```diff
@@ -7,6 +7,7 @@ $mat-button-toggle-padding: 0 16px !default;
 $mat-button-toggle-height: 36px !default;
 $mat-button-toggle-border-radius: 2px !default;
 
+:not(.mat-button-toggle-group) > .mat-button-toggle,
```

Comment:
Could we do this instead by having a css class like `mat-button-toggle-standalone` that is applied via host binding
```ts
host: {
  '[class.mat-button-toggle-standalone]': 'buttonToggleGroup',
}
```

That way the styles will have a much lower specificity and will be easier to override